### DISCUSSION TITLE
network/modules/ingress-nginx/scope/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3594] [Announcements]
# network/modules/ingress-nginx/scope/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/network/modules/ingress-nginx/scope/

**Comment by muma378** [comment]
文档有误，和IngressClass的参数没有关系
